hack/build-images.sh

```diff
@@ -7,7 +7,7 @@ set -o pipefail
 STARTTIME=$(date +%s)
 source_root=$(dirname "${0}")/..
 
-prefix="openshift/origin-"
+prefix="docker.io/openshift/origin-"
```

Comment:
@jcantrill @smarterclayton This breaks local builds and we need to get this resolved.